Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -454,6 +476,49 @@ public struct TargetSourcesBuilder {
 
         return contents
     }
+
+    public static func computeContents(for generatedFiles: [AbsolutePath], toolsVersion: ToolsVersion, additionalFileRules: [FileRuleDescription], defaultLocalization: String?, targetName: String, targetPath: AbsolutePath, observabilityScope: ObservabilityScope) -> (sources: [AbsolutePath], resources: [Resource]) {
+        var sources = [AbsolutePath]()
+        var resources = [Resource]()
+
+        generatedFiles.forEach { absPath in
+            // 5.6 handled treated all generated files as sources.
+            if toolsVersion <= .v5_6 {
```

Comment:
guard?

Reply:
Feels wrong to use `guard` for a legitimate case?